src/main/java/io/github/jhipster/async/ExceptionHandlingAsyncTaskExecutor.java

```diff
@@ -71,7 +73,7 @@ private Runnable createWrappedRunnable(final Runnable task) {
     }
 
     protected void handle(Exception e) {
-        log.error("Caught async exception", e);
+        log.error(EXCEPTION_MESSAGE, e);
```

Comment:
I'm not a fan of using constants for log messages unless they are reused in many places. I see that you use them for test assertions as well but testing log messages seems a bit too much to me @jdubois  what do you think?

Reply:
Yes, that’s why I made them constants. Could be made package private if you prefer. Or like you say maybe asserting log messages is too much, but for what it’s worth I actually quite like it. Especially for failure messages, logging is an important function and as such ought to be tested.

Reply:
Oh and also, the way the log testing works also keeps the console nice and clean during `mvn test`. So expected exceptions during tests, for example, don’t spew out endless traces.

Reply:
I'd rather also have normal Strings as they are more readable, but that's not a huge issue